src/main/java/uk/gov/digital/justice/hmpps/sentenceplan/jpa/entity/ActionEntity.java

```diff
@@ -27,52 +27,37 @@
 
     private ActionStatus status;
 
-    private List<UUID> needs;
+    private List<UUID> needs = new ArrayList<>(0);
 
     private String intervention;
 
     private int priority;
 
-    private List<ProgressEntity> progress;
+    private List<ProgressEntity> progress = new ArrayList<>(0);
 
-    private LocalDateTime created;
+    private LocalDateTime created = LocalDateTime.now();
 
     private LocalDateTime updated;
 
-
-    public ActionEntity() {
-        this.progress = new ArrayList<>(0);
-    }
-
     public ActionEntity(List<ActionOwner> owner, String ownerOther, String description, String strength, ActionStatus status, List<UUID> needs, String intervention) {
-        var now = LocalDateTime.now();
         this.id = UUID.randomUUID();
-        this.owner = new ArrayList<>(0);
-        this.progress = new ArrayList<>(0);
-        this.created = now;
-        this.updated = now;
         update(owner, ownerOther, description, strength, status, needs, intervention);
+        this.updated = this.created;
     }
 
     public void updateAction(List<ActionOwner> owner, String ownerOther, String description, String strength, ActionStatus status, List<UUID> needs, String intervention) {
         update(owner, ownerOther, description, strength, status, needs, intervention);
         this.updated = LocalDateTime.now();
     }
 
-    @JsonIgnore
```

Comment:
Because the comment will be moved from the Action to the Objective we are updating the Objective when we progress a step so we should use that updated date and it means we can get rid of this expensive call.